src/pages/ElasticLegacy/FarmLegacy.tsx

```diff
@@ -0,0 +1,373 @@
+        const gas = await library.getSigner().estimateGas({
+          to: claimInfo.address,
+          data: claimInfo.encodedData,
+        })
+
+        const { hash } = await library.getSigner().sendTransaction({
+          to: claimInfo.address,
+          data: claimInfo.encodedData,
+          gasLimit: calculateGasMargin(gas),
+        })
+
+        setAttemptingTxn(false)
+        setTxHash(hash || '')
+
+        addTransactionWithType({
+          hash,
+          type: TRANSACTION_TYPE.CLAIM_REWARD,
+          extraInfo: {
+            summary: 'farming rewards',
+          },
+        })
+      } catch (e) {
+        setAttemptingTxn(false)
+        setErrorMessage(e?.message || JSON.stringify(e))
+      }
+    }
+  }
+
+  const farmContract = useProMMFarmContract(config[chainId].farmContract || '')
+
+  const handleWithdraw = async () => {
+    setShowConfirmModal('withdraw')
+    if (!farmContract) {
+      setErrorMessage('No contract found')
+      return
+    }
+
+    try {
+      const nftIds = farmPositions.map(item => item.id)
+      const estimateGas = await farmContract.estimateGas.emergencyWithdraw(nftIds)
+      const tx = await farmContract.emergencyWithdraw(nftIds, {
+        gasLimit: calculateGasMargin(estimateGas),
+      })
+      setAttemptingTxn(false)
+      setTxHash(tx.hash || '')
+
+      addTransactionWithType({
+        hash: tx.hash,
+        type: TRANSACTION_TYPE.ELASTIC_FORCE_WITHDRAW_LIQUIDITY,
+        extraInfo: { contract: config[chainId].farmContract },
+      })
+    } catch (e) {
+      setAttemptingTxn(false)
+      setErrorMessage(e?.message || JSON.stringify(e))
+    }
+  }
+
+  return (
+    <Wrapper>
+      <Text>
+        We paused all our Elastic farms on 18 April 2023, 4pm UTC. Information on this can be found in our announcement{' '}
+        <ExternalLink href="https://twitter.com/KyberNetwork/status/1648265647858790400?s=20">here</ExternalLink>
+      </Text>
+
+      {!!numberOfPosition && claimInfo ? (
+        <Text>
+          You still have{' '}
+          <Text color={theme.warning} as="span">
+            {numberOfPosition} liquidity positions
+          </Text>{' '}
+          and{' '}
+          <Text color={theme.warning} as="span">
+            {formatDollarAmount(unclaimedUSD)} unclaimed farming rewards
+          </Text>{' '}
+          that you haven&apos;t withdrawn from the Elastic farms yet. You will first need to withdraw all your liquidity
+          positions by clicking on ‘Withdraw Positions’ button before you can claim your reward by clicking the
+          &apos;Claim Rewards&apos; button{' '}
+        </Text>
+      ) : !!numberOfPosition ? (
+        <Text>
+          You still have{' '}
+          <Text as="span" color={theme.warning}>
+            {numberOfPosition} liquidity positions
+          </Text>{' '}
+          that you haven&apos;t withdrawn from the Elastic farms yet, please withdraw and remove your funds on Elastic
+          as soon as possible
```

Comment:
missing trans

Reply:
Legacy page should be remove in the future, no translation for now